src/test/suite/composite-escape.test.ts

```diff
@@ -13,78 +13,99 @@ import {
 
 describe("Composite escape key", () => {
     let client: NeovimClient;
+    let doc;
     before(async () => {
         client = await attachTestNvimClient();
+        doc = await vscode.workspace.openTextDocument({
+            content: "",
+        });
+        await vscode.window.showTextDocument(doc);
+        await wait(1000);
     });
     after(async () => {
-        await closeNvimClient(client);
-    });
-
-    afterEach(async () => {
         await closeAllActiveEditors();
+        await closeNvimClient(client);
     });
 
-    it("Works", async () => {
-        const doc = await vscode.workspace.openTextDocument({
-            content: "",
-        });
-        await vscode.window.showTextDocument(doc);
-        await wait(1000);
+    beforeEach(async () => await sendVSCodeKeys("S"));
 
-        await sendVSCodeKeys("i");
-        await vscode.commands.executeCommand("vscode-neovim.compositeEscape1", "j");
-        await wait(300);
-        await vscode.commands.executeCommand("vscode-neovim.compositeEscape1", "j");
+    it("escapes on jk and removes 'j'", async () => {
+        await vscode.commands.executeCommand("vscode-neovim.compositeEscape", { key: "j" });
+        await wait(50);
+        await vscode.commands.executeCommand("vscode-neovim.compositeEscape", { key: "k" });
+        await wait(69);
         await assertContent(
             {
-                mode: "i",
+                content: [""],
```

Comment:
I wonder whether it would be more robust to have actual content here, to test you don't accidentally delete more than just one letter/delete one to the left/right instead? i.e. start with content abc, and ensure that's what's left at the end?

Reply:
good call -- do you know of a quick way to run just the tests in this file, or is modifying line 21 of in `index.ts` the only way?

`        glob("**/**.test.js", { cwd: testsRoot }, (err, files) => {`
to
`        glob("**/<filename>.test.js", { cwd: testsRoot }, (err, files) => {`